crates/core/tedge/Cargo.toml

```diff
@@ -24,7 +24,6 @@ reqwest = { version = "0.11", default-features = false, features = ["blocking",
 rpassword = "5.0"
 rumqttc = "0.10"
 rustls = "0.20.2"
-rustls_0_19 = {package = "rustls", version = "0.19.0" }
```

Comment:
Does `tedge` still depend on `rustls`? Why?

Reply:
Yes, its needed, because its used in `get_webpki_error_from_reqwest(err: reqwest::Error)`